### PR TITLE
Change query populating payments to filter out entries older than 18 …

### PR DESCRIPTION
**Pull request by ivan-unfolds**
…months

### CODE REVIEW

File: api/controllers/MembersController.js [hunk 1]
```diff
@@ -19,7 +19,11 @@ module.exports = {
   accountInfo: function (req, res) {
     Members
       .findOne(req.session.user.id)
-      .populateAll()
+      .populate('events_booked')
+      .populate('membership_type')
+      .populate('payments', {
+        where: { date: { '>': new Date(Date.now() - 47340000000) }}
```
Comment (rjmk) [inline api/controllers/MembersController.js]: Let's replace `47340000000` with `1.5 * 365 * 24 * 60 * 60`

**Comment by rjmk** [comment]
I'm assuming the 2 years was on purpose

**Comment by ivan-unfolds** [comment]
@rjmk yeah it was, Richard asked for it on pivotal tracker. I was thinking of going for 365.25 as well, relive the good old times of my physics degree. But then I thought about floating points 😨

**Comment by rjmk** [comment]
@ivanmauricio Hahaha

**Comment by ivan-unfolds** [comment]
Also as I changed from `populateAll()` we are now not getting `activation_codes` and `reset_password_codes` from DB. Is that okay?

**Comment by rjmk** [comment]
Yeah, fine

**Pull request merged**